### ISSUE TITLE
command list: python

### ISSUE THREAD

**Issue by mwcz** [CommandLineHeroes/clh-bash#8]
Get all python:

- keywords
- global standard library functions (like `max()`, but not ones inside modules like `math.sin()`)

**Comment by Jared-Sprague** [comment]
Making this high-priority since python is the language of choice for sysadmins and devops, which will be a lot of the target demo

**Comment by mwcz** [comment]
Sounds good, but I want to point out that bash and python are high priority now, and HTML+JS are low priority.  I set priorities so that we'd have one back end lang and one front end lang.  If you like python as the sysadmin language, we should downgrade bash to make room for HTML or JS.  Since the goal of adding more languages was to appeal to more demographics, having two sysadmin langs as the top priority isn't achieving that goal.

**Comment by Jared-Sprague** [comment]
Oh I thought js was high priority.  It should be.  Only html should be low

**Comment by MikeFindsThings** [comment]
I think I can do this tonight.

**Comment by Jared-Sprague** [comment]
@OpSecIsSexy  Thanks for the PR!  Can you describe your process for generating the list?

**Comment by MikeFindsThings** [comment]
Absolutely. First I found a list online of the keywords and built in functions used in Python. I took this list and made it into a list in Python. Then I used a function to loop through each item in the list and format it to match the middle section of the appropriate JS data type (sorry, not familiar with JS so I don't know the name, just copied the format of one that already existed in the cmds folder), and saved that in a txt document. The opening and ending brackets as well as the comments were entered manually, and the body of the list was copied and pasted from the txt.
